### PR TITLE
[Feature] remove EVM: 1. add cita-vm. [skip ci]

### PR DESCRIPTION
**Pull request by leeyr338**
1. add cita-vm.

### CODE REVIEW

File: cita-executor/core/src/cita_executive.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// CITA
```
Comment (kaikai1024) [inline cita-executor/core/src/cita_executive.rs]: No need to add this.
Change the GPL part if you want to.

Comment (ouwenkg) [reply cita-executor/core/src/cita_executive.rs]: All these license docs could be removed after our task.

Comment (kaikai1024) [reply cita-executor/core/src/cita_executive.rs]: Then, why do we add it first?

Comment (ouwenkg) [reply cita-executor/core/src/cita_executive.rs]: Adding now or Removing together later is accecpted to me.

Comment (leeyr338) [reply cita-executor/core/src/cita_executive.rs]: okay, delete it.

**Pull request merged**